src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/ConsensusRuleUnitTestBase.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+using Stratis.Bitcoin.Features.Consensus.Rules;
+using Stratis.Bitcoin.Features.Consensus.Rules.CommonRules;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests.Rules
+{
+    public class ConsensusRuleUnitTestBase
+    {
+        protected Network network;
+        protected Mock<ILoggerFactory> loggerFactory;
+        protected Mock<IDateTimeProvider> dateTimeProvider;
+        protected ConcurrentChain concurrentChain;
+        protected NodeDeployments nodeDeployments;
+        protected ConsensusSettings consensusSettings;
+        protected Mock<ICheckpoints> checkpoints;
+        protected List<ConsensusRule> ruleRegistrations;
+        protected Mock<IRuleRegistration> ruleRegistration;
+        protected TestConsensusRules consensusRules;
+
+        public ConsensusRuleUnitTestBase()
+        {
+            Block.BlockSignature = false;
+            Transaction.TimeStamp = false;
+            this.network = Network.TestNet;
+            this.loggerFactory = new Mock<ILoggerFactory>();
+            this.loggerFactory.Setup(l => l.CreateLogger(It.IsAny<string>()))
+                .Returns(new Mock<ILogger>().Object);
+            this.dateTimeProvider = new Mock<IDateTimeProvider>();
+
+            this.concurrentChain = new ConcurrentChain(this.network);
+            this.nodeDeployments = new NodeDeployments(this.network, this.concurrentChain);
+            this.consensusSettings = new ConsensusSettings
+            {
+                BlockAssumedValid = null,
+                UseCheckpoints = true
```

Comment:
I wonder if this should be true by default

Reply:
Fair enough. I will move that out. I will move that out and fix the test that needs it.